src/test/java/com/scalar/cassy/transferer/AzureBlobFileUploaderTest.java

```diff
@@ -44,8 +50,10 @@
   private static final String ANY_STOREBASE_URI = "container_name";
   private static final Joiner joiner = Joiner.on("/").skipNulls();
   private static final FileSystem fs = FileSystems.getDefault();
-  @Mock private BlobContainerAsyncClient containerClient;
-  @Mock private BlobAsyncClient blobClient;
+  @Mock private BlobContainerClient containerClient;
+  @Mock private BlobClient blobClient;
+  @Mock private Future future;
```

Comment:
It's never used.

Reply:
Thank you for your review comments.
Unused variables and imports removed f8d33d6 and 8f1871186832d28ea8c0f0b548ec779b3aeedd9b